flax/linen/transforms.py

```diff
@@ -780,7 +780,8 @@ def body_fn(cls, carry, xs, *args, **kwargs):
 
   Args:
     target: a ``Module`` or a function taking a ``Module``
-      as its first argument.
+      as its first argument. Scanning happens from the third and all
```

Comment:
Perhaps something like: The scanned function receives the module (self) and carry as the first two arguments. The remainder of the arguments are scanned over

Or any other phrasing that makes clear what the first two args are